sources/packages/web/src/components/common/SupportingUserForm.vue

```diff
@@ -117,6 +119,7 @@ export default defineComponent({
           hasValidSIN: supportingUser.value.personalInfo?.hasValidSIN,
           parentFullName: supportingUser.value.parentFullName,
         };
+        isDataReady.value = true;
```

Comment:
As a general rule, it should be set to `false` before the API call to allow it to react to the data being reloaded.
I am not sure if we should have it for this form, but for forms that potentially reload data, we can have it.

Reply:
Added the setting to false before API call. Even if the code to load the data is moved to a separate method it will move along.